### PR TITLE
e2e: reorder cleanup

### PR DESCRIPTION
**Pull request by mrogers950**
Move all resource cleanup to cleanUp(). This gives us more control over ordering.

Fixes https://github.com/openshift/file-integrity-operator/issues/31

### CODE REVIEW

File: tests/e2e/helpers.go [hunk 1]
```diff
@@ -61,12 +86,7 @@ func setupTestRequirements(t *testing.T) *framework.TestCtx {
 	return framework.NewTestCtx(t)
 }
 func setupFileIntegrityOperatorCluster(t *testing.T, ctx *framework.TestCtx) {
-	cleanupOptions := framework.CleanupOptions{
-		TestContext:   ctx,
-		Timeout:       cleanupTimeout,
-		RetryInterval: cleanupRetryInterval,
-	}
-	err := ctx.InitializeClusterResources(&cleanupOptions)
```
Comment (JAORMX) [inline tests/e2e/helpers.go]: Why did you remove the cleanUp options?

Comment (mrogers950) [reply tests/e2e/helpers.go]: So that the resource setup part does not add any cleanup actions. As far as I could tell, there is not a way to enforce an order during its cleanup so I handle it all in cleanUp().

**Comment by JAORMX** [comment]
Added a comment in the issue #31

**Comment by mrogers950** [comment]
@JAORMX I've revised this PR to implement your suggestion. Now the aide-clean deployment is run at the end of each of the test functions, but before the rest of the cleanup. The event logs from CI should indicate if this helps.

I also added in a fix to the error logging to make it easier to tell where it fails (I was testing with this same fix in #28).

**Comment by JAORMX** [comment]
This solution looks less invasive. I dig it.

**Comment by JAORMX** [comment]
```
 /bin/bash: line 286: CLUSTER_NETWORK_MANIFEST: unbound variable 
```

huh....

**Comment by mrogers950** [comment]
> ```
>  /bin/bash: line 286: CLUSTER_NETWORK_MANIFEST: unbound variable 
> ```
> 
> huh....

This was reported to be fixed.
/retest

**Comment by mrogers950** [comment]
We got 
```
 === RUN   TestFileIntegrityConfigurationIgnoreMissing
--- FAIL: TestFileIntegrityConfigurationIgnoreMissing (62.50s)
    client.go:57: resource type ServiceAccount with namespace/name (openshift-file-integrity/file-integrity-operator) created
    client.go:57: resource type Role with namespace/name (openshift-file-integrity/file-integrity-operator) created
    client.go:57: resource type Role with namespace/name (openshift-file-integrity/scc-priv) created
    client.go:57: resource type RoleBinding with namespace/name (openshift-file-integrity/file-integrity-operator) created
    client.go:57: resource type RoleBinding with namespace/name (openshift-file-integrity/scc-priv) created
    client.go:57: resource type Deployment with namespace/name (openshift-file-integrity/file-integrity-operator) created
    helpers.go:67: Initialized cluster resources
    wait_util.go:64: Waiting for full availability of file-integrity-operator deployment (0/1)
    wait_util.go:64: Waiting for full availability of file-integrity-operator deployment (0/1)
    wait_util.go:64: Waiting for full availability of file-integrity-operator deployment (0/1)
    wait_util.go:70: Deployment available (1/1)
    e2e_test.go:78: Creating FileIntegrity object
    client.go:57: resource type  with namespace/name (openshift-file-integrity/test-check) created
    e2e_test.go:170: FileIntegrity ready (Active)
    e2e_test.go:170: FileIntegrity ready (Initializing)
    e2e_test.go:256: status changed to initialization in error
    e2e_test.go:170: FileIntegrity ready (Active) 
```
Retesting to see if this flaked or is a persistent failure.

/retest

**Comment by mrogers950** [comment]
Another green, once more to be sure.
/test e2e-aws

**Comment by mrogers950** [comment]
@JAORMX this should be all set.
